lib/shape.ml

```diff
@@ -38,20 +38,20 @@ let render_shape s =
     let b = denormalize line.b in
     draw_line a.x a.y b.x b.y
 
-let circle ?x ?y r =
-  match (x, y) with
-  | Some x, Some y -> Circle { c = { x; y }; radius = r }
-  | _ -> Circle { c = { x = 0; y = 0 }; radius = r }
+  let circle ?x ?y ?fill ?stroke r =
+    match (x, y) with
+    | Some x, Some y -> Circle { c = { x; y }; radius = r; fill; stroke }
+    | _ -> Circle { c = { x = 0; y = 0 }; radius = r; fill; stroke }  
 
-let rectangle ?x ?y length width =
+let rectangle ?x ?y ?fill ?stroke length width =
```

Comment:
We can use default values with, e.g., `?(fill = Transparent)` and `?(stroke = black)`